test/CDB_StatsTest.sql

```diff
@@ -0,0 +1,16 @@
+-- continuous uniform distribution has kurtosis = -6/5, skewness = 0.0
+-- http://mathworld.wolfram.com/UniformDistribution.html
+set client_min_messages to ERROR;
+
+With dist As (
+  SELECT random()::numeric As val
+  FROM generate_series(1,5000000) t
```

Comment:
Can we test with a lower range? Testing with such a big one means the test is very slow.

What is the minimum range we can use to validate these functions?